app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
Hm, that feels scary to me but I'm not sure why. I guess we could put it in and smoke test it, but the idea of a whole mess of new events going from `event.discussion.present? == false` to `event.discussion.present? == true` made me afraid.

The new_discussion event is related to the bug I'm fixing because it shows up in the eventsView. When I call $scope.discussion.events(), I get a new_discussion event back. Which would be ok if it had a nil sequenceId (or a sequenceId of 0, which this fix does), but is not okay if it has a sequenceId of 40,000 (and it's the only event), because it then goes "Wow, the first sequenceId is over 9,000! We must have previous content to load, show the load previous content button'

Oh, I should mention somewhere that this fix isn't backwards compatible, we'll need to run an update on existing 'new_discussion' events to get it to fix old threads.